actors/miner/tests/util.rs

```diff
@@ -843,11 +843,14 @@ impl ActorHarness {
         let mut all_recovered = BitField::new();
         let dln = self.get_deadline(rt, deadline.index);
         for p in &params.partitions {
-            let partition = dln.load_partition(&rt.store, p.index).unwrap();
-            let expected_faults = &partition.faults - &partition.recoveries;
-            let skipped = get_bitfield(&p.skipped);
-            all_ignored |= &(&expected_faults | &skipped);
-            all_recovered |= &(&partition.recoveries - &skipped);
+            let maybe_partition = dln.load_partition(&rt.store, p.index);
+            if maybe_partition.is_ok() {
```

Comment:
Why are we not panicing if this fails? That sounds like something we should be asserting.

Reply:
I think this is to allow bad partition indexes into params to exercise bad inputs failing in the actual method instead of the test setup.

Reply:
We want to test invalid submissions reaching the miner actor itself.